### PR TITLE
Remove logger proxy from exceptions

### PR DESCRIPTION
**Pull request by insacuri**
# Motivation and Context
To stop passing logger proxy via exceptions in frontstage for code consistency

# What has changed
Refactoring of logger

# How to test?

# Links

# Screenshots (if appropriate):

### CODE REVIEW

File: frontstage/controllers/case_controller.py [hunk 1]
```diff
@@ -45,10 +45,8 @@ def get_case_by_case_id(case_id):
     try:
         response.raise_for_status()
     except requests.exceptions.HTTPError:
-        raise ApiError(logger, response,
-                       case_id=case_id,
-                       log_level='warning' if response.status_code == 404 else 'exception',
-                       message='Failed to retrieve case by case id')
+        logger.error('Failed to retrieve case by case id', case_id=case_id, )
```
Comment (mfcrocker) [inline frontstage/controllers/case_controller.py]: This line just needs cleaning up a little bit - should end `case_id=case_id)`

File: frontstage/controllers/party_controller.py [hunk 2]
```diff
@@ -145,7 +139,8 @@ def resend_verification_email(party_id):
         response.raise_for_status()
     except requests.exceptions.HTTPError:
         logger.exception('Re-sending of verification email failed', party_id=party_id)
-        raise ApiError(logger, response, log_level='exception', message='Re-sending of verification email failed')
+        logger.error('Re-sending of verification email failed')
```
Comment (insacuri) [inline frontstage/controllers/party_controller.py]: This isn't needed.  The logger.exception line above does basically the same thing

**Comment by insacuri** [comment]
I can't request changes cause I raised the PR >_< . I've only got 1 comment and I'll take 1 more quick look next week but it's basically there!

File: .python-version [hunk 3]
```diff
@@ -1 +1 @@
-3.6.8
\ No newline at end of file
+3.6.9
```
Comment (insacuri) [inline .python-version]: Really minor sorry, but can you add `.python-version` to the .gitignore file and delete this from the PR

**Comment by insacuri** [comment]
Looks good 👍

**Pull request merged**